etc/ProdOfflineConfiguration.py

```diff
@@ -536,6 +535,13 @@
 
 DATASETS = ["ReservedDoubleMuonLowMass"]
 
+for dataset in DATASETS:
+    addDataset(tier0Config, dataset,
+               do_reco=False,
+               scenario=ppScenario)
+
+DATASETS = ["ParkingSingleMuon"]
+
 for dataset in DATASETS:
     addDataset(tier0Config, dataset,
                do_reco=False,
```

Comment:
@drkovalskyi I understood from @silviodonato that you know how this new parking PD should be configured (same as the other parking datasets AFAIU), could you suggest it here?

Reply:
I think that's the case. Dima wont be able to properly reply until Monday

Reply:
Ok so maybe we can leave it as is (i.e. only RAW output) and adjust it next week together with the rest?